### PR TITLE
[2.0] stop building docker-images for 2.0 release branch.

### PR DESCRIPTION
**Pull request by scv119**
## Why are these changes needed?

stop building docker image for 2.0.0 branch so we can cherry-pick document changes to 2.0.0 release branch without generating/uploading new docker images.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by rkooo567 — APPROVED** [review]
LGTM, but I will also wait for @simon-mo's approval

**Review by simon-mo — APPROVED** [review]
LGTM. We should just put this into the release handbook. A hood step to have.

**Pull request merged**
